### PR TITLE
Added options: head/code shadows, py3doc code, hilight sections, linkcolor, visitedlinkcolor

### PR DESCRIPTION
**Pull request by joelfrederico**
I noticed that a bunch of the html_theme_options don't work, so I fixed a few to make it possible to tweak the theme closer to how the python 3 documentation looks. I use these html_theme_options copied from python's repository (some of which, I'm sure, are redundant):

```
html_theme_options = {
    'bodyfont': '\'Lucida Grande\', Arial, sans-serif',
    'headfont': '\'Lucida Grande\', Arial, sans-serif',
    'footerbgcolor': 'white',
    'footertextcolor': '#555555',
    'relbarbgcolor': 'white',
    'relbartextcolor': '#666666',
    'relbarlinkcolor': '#444444',
    'sidebarbgcolor': 'white',
    'sidebartextcolor': '#444444',
    'sidebarlinkcolor': '#444444',
    'bgcolor': 'white',
    'textcolor': '#222222',
    'linkcolor': '#0072AA',
    'visitedlinkcolor': '#6363bb',
    'headtextcolor': '#1a1a1a',
    'headbgcolor': 'white',
    'headlinkcolor': '#aaaaaa',
    'headshadow': False,
    'codeshadow': False,
    'newstylecode': False,
    'highlightcurrent': False
    }
```

### CODE REVIEW

**Comment by ionelmc** [comment]
Seems like a very reasonable change. Thanks!

File: src/sphinx_py3doc_enhanced_theme/static/pydoctheme.css_t [hunk 1]
```diff
@@ -284,13 +288,17 @@ li > img {
...
 h1, h2, h3, h4, h5, h6 {
+    {% if theme_headshadow %}
     text-shadow: 1px 1px 2px #ddd;
+    {% endif %}
     font-weight: bold;
     color: #333;
 }
 code, tt {
     background: none;
+    {% if codeshadow %}
```
Comment (ionelmc) [inline src/sphinx_py3doc_enhanced_theme/static/pydoctheme.css_t:299]: @joelfrederico Shouldn't this be theme_codeshadow? Can you make another PR? :grin:

Comment (joelfrederico) [inline src/sphinx_py3doc_enhanced_theme/static/pydoctheme.css_t:299]: Ah whoops. I also just found your CONTRIBUTING.rst... muh bad... I don't think I followed any of your recommendations.

Anyways, PR coming up! Totally right, also my bad.
